### ISSUE TITLE
Community 23.x compatibility

### ISSUE THREAD

**Issue by sergiobaiao** [jottley/alfresco-maxversion-policy#12]
Hi there, 

Is it possible to make this compatible to ACS Community 23.x?

**Comment by jottley** [comment]
Yes, this would be possible.  I'll try to set some time aside to do this this weekend.  There maybe a PR already to do this...

**Comment by sergiobaiao** [comment]
hi jottley, any news about this?
